src/entity/Human.php

```diff
@@ -191,8 +195,13 @@ public function getHungerManager() : HungerManager{
 	}
 
 	public function consumeObject(Consumable $consumable) : bool{
```

Comment:
Can be simplified to have only two returns instead of duplicating the call to the parent